### PR TITLE
no unused params: actions

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: shared/actions/team-building.tsx [hunk 1]
```diff
@@ -52,7 +52,8 @@ function* searchResultCounts(state: TypedState, {payload: {namespace}}: NSAction
     .filter(s => s !== teamBuildingSelectedService && s !== 'contact')
     .filter(s => !teamBuildingState.teamBuildingSearchResults.hasIn([teamBuildingSearchQuery, s]))
 
-  const isStillInSameQuery = (state: TypedState): boolean =>
+  // TODO this isn't reading from state????? how can it change. likely wrong
+  const isStillInSameQuery = (_: TypedState): boolean =>
```
Comment (chrisnojima) [inline shared/actions/team-building.tsx]: @MarcoPolo this seems wrong

Comment (MarcoPolo) [reply shared/actions/team-building.tsx]: Oh, good catch!

Comment (MarcoPolo) [inline shared/actions/team-building.tsx]:
```suggestion
  const isStillInSameQuery = (state: TypedState): boolean =>
    state[namespace].teamBuilding.teamBuildingSearchQuery === teamBuildingSearchQuery &&
    state[namespace].teamBuilding.teamBuildingSelectedService === teamBuildingSelectedService
    
```

**Comment by MarcoPolo** [comment]
Pushed the fix

**Pull request merged**